eZ/Publish/Core/FieldType/DateAndTime/Type.php

```diff
@@ -148,7 +148,9 @@ protected function getSortInfo(BaseValue $value)
     /**
      * Converts an $hash to the Value defined by the field type.
      *
-     * @param mixed $hash Null or associative array containing timestamp and optionally date in RFC850 format.
+     * @param mixed $hash Null or associative array containing timestamp and optionally date in RFC850 format,
```

Comment:
Shouldn't we document the keys that can be used in the hash more accurately ?

It will also be needed to update the fieldtype doc, as we try to document what the hashes and input values can be like.